tests/Unit/FeedXMLGenerationTest.php

```diff
@@ -389,6 +399,55 @@ public function testPropertyAdditionalImageLinkXML() {
 		$this->assertEquals( '<g:additional_image_link><![CDATA[http://example.org/wp-content/uploads/product_image_1.png,http://example.org/wp-content/uploads/product_image_2.png]]></g:additional_image_link>', $xml );
 	}
 
+	/**
+	 * @group feed
+	 */
+	public function testAttributesConditionXML() {
+		$method = ( new ReflectionClass( ProductsXmlFeed::class ) )->getMethod( 'get_attributes_xml' );
+		$method->setAccessible( true );
+
+		$product = WC_Helper_Product::create_simple_product();
+		$xml = $method->invoke( null, $product, '\t' );
```

Comment:
With single quotes the indent won't get interpreted as a tab. Although in this test we can probably leave it as an empty string.